ui/packages/lib/src/components/ErrorBoundary.js

```diff
@@ -24,31 +23,46 @@ export class ErrorBoundary extends React.Component {
 
   render() {
     return !!this.state.errorInfo ? (
-      <EuiEmptyPrompt
-        iconType="editorStrike"
-        title={<h2>Something Went Wrong</h2>}
-        body={
-          <Fragment>
-            <p>
-              Something wen't wrong with the UI, no worries, we will fix this as
-              soon possible
-            </p>
-            <p>Please report this to maintainer my friend</p>
-          </Fragment>
-        }
-        actions={
-          <EuiButton
-            color="primary"
-            fill
-            onClick={() =>
-              Sentry.showReportDialog({ eventId: this.state.eventId })
-            }>
-            Report Feedback
-          </EuiButton>
-        }
-      />
+      <EuiPageTemplate>
+        <EuiPageTemplate.EmptyPrompt
+          color="subdue"
+          iconType="editorStrike"
+          title={<h2>Something Went Wrong</h2>}
+          body={
+            <>
+              <p>
+                Something went wrong with the UI, no worries, we will fix
+                <br />
```

Comment:
Why the extra `<br />` here?

Reply:
To explicitly break the message into two lines. In other words, to avoid situation, where we have something like:
```
Something went wrong with the UI, no worries, we will fix this as soon as
                               possible
```